### PR TITLE
Basic app up and running

### PR DESCRIPTION
**Pull request by tenshiemi**

### CODE REVIEW

File: client/.babelrc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "browsers": ["last 2 versions", "safari >= 7"]
```
Comment (ferlatte) [inline client/.babelrc:5]: We don't have to support safari back that far?

File: client/package.json [hunk 2]
```diff
@@ -1,18 +1,40 @@
...
+    "path": "^0.12.7",
+    "webpack": "^1.13.2",
+    "webpack-dev-server": "^1.16.2"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:trussworks/Truss-PQVP.git"
+  },
+  "author": "",
+  "license": "MIT"
```
Comment (ferlatte) [inline client/package.json:39]: Our license is not MIT.

**Comment by ferlatte** [comment]
Please update the README with instructions on how to develop and deploy. Also, I didn't see any changes to Circle CI; will this deploy correctly?

File: client/.babelrc (hunk 1 above)
Comment (tenshiemi) [reply client/.babelrc:5]: This is just temporary, I'll update it

**Pull request closed without merging**